.github/scripts/sql-correctness/get-dolt-correctness-job-json.sh

```diff
@@ -41,6 +41,7 @@ echo '
               { "name": "REPO_ACCESS_TOKEN", "value": "'$REPO_ACCESS_TOKEN'"},
               { "name": "ACTOR", "value": "'$ACTOR'"},
               { "name": "ACTOR_EMAIL", "value": "'$ACTOR_EMAIL'"}
+              { "name": "DOLT_DEFAULT_BIN_FORMAT", "value": "'NOMS_BIN_FORMAT'"}
```

Comment:
this doesn't need to be updated in the other template?

Reply:
it's handled differently in the other benchmarks